app/mailers/subscription_mailer.rb

```diff
@@ -1,7 +1,7 @@
 # encoding: utf-8
 
 class SubscriptionMailer < ActionMailer::Base
-  default from: 'Otvorené Súdy <noreply@otvorenesudy.sk>', css: 'mailers/layout'
+  default from: "#{t('open_courts')} <noreply@otvorenesudy.sk>", css: 'mailers/layout'
```

Comment:
this causes tests and entire application in production mode to fail, since `t` is not a method defined on class level of mailer...